themes/bootstrap3/templates/RecordDriver/SolrOverdrive/status-full.phtml

```diff
@@ -17,26 +17,38 @@ foreach ($ids as $id) {
     $current = [];
     $current['status'] = 'OK';
     $current['id'] = $id;
-    $avail = $this->result->data[$id] ?? new \stdClass();
+    $avail = $this->result->data[$id] ?? (object)['code' => ''];
+    $current['full_status'] = '';
 
     if ($noshow) {
-        $current['full_status'] = '';
-    } elseif ($isError || $avail->code == 'od_code_resource_not_found') {
+        //patron does not have access to overdrive. don't show avail
+    } elseif ((isset($avail->code) && $avail->code == 'od_code_resource_not_found') || $isError) {
         $current['error'] = $this->transEsc('status_unknown_message');
-        $current['full_status'] = '';
-    } elseif ($loginForAvail || $avail->code == 'od_code_login_for_avail') {
+    } elseif ($loginForAvail) {
         $current['full_status'] = "<p class='alert-info'>"
             . $this->transEsc('od_code_login_for_avail') . '</p>';
     } else {
-        $current['full_status'] = "<p><div class='copies'><strong>"
-            . $this->transEsc('od_avail_total') . '</strong> '
-            . $this->escapeHtml($avail->copiesOwned)
-            . "</div><div class='avail'><strong>"
-            . $this->transEsc('od_avail_avail') . '</strong> '
-            . $this->escapeHtml($avail->copiesAvailable)
-            . "</div><div class='holds'><strong>"
-            . $this->transEsc('od_avail_holds') . '</strong> '
-            . $this->escapeHtml($avail->numberOfHolds) . '</div></p>';
+        $current['full_status'] .= '<div class="availability"><p>';
+        if ($avail->copiesAvailable > 0) {
+            $current['full_status'] .= '<span class="text-success">';
+            $current['full_status'] .= $this->icon('status-available', 'icon-link__icon');
+            $current['full_status'] .= '</span> ';
+        } else {
+            $current['full_status'] .= '<span class="text-danger">';
+            $current['full_status'] .= $this->icon('status-unavailable', 'icon-link__icon');
+            $current['full_status'] .= '</span> ';
+        }
+        if ($avail->availabilityType == 'AlwaysAvailable') {
+            $current['full_status'] .= $this->transEsc('od_unlimited');
+        } else {
+            $current['full_status'] .= $avail->copiesAvailable . ' / ' . $avail->copiesOwned;
+        }
+        $current['full_status'] .= ' ' . $this->transEsc('od_copies_available');
```

Comment:
Rather than concatenating strings here, it would probably be better to change the od_copies_available string to something like `%%copies%% copies available` and then change this code to something like:

```
$copies = $avail->availabilityType == 'AlwaysAvailable'
    ? $this->transEsc('od_unlimited');
    : ($avail->copiesAvailable . ' / ' . $avail->copiesOwned);
$current['full_status'] .= ' ' . $this->transEsc('od_copies_available', ['%%copies%%' => $copies]);
```

...and then languages that put things in different orders can adjust things appropriately in translation.